contributing/documentation/overview.rst

```diff
@@ -107,6 +107,12 @@ your project:
     upstream  https://github.com/symfony/symfony-docs.git (fetch)
     upstream  https://github.com/symfony/symfony-docs.git (push)
 
+Fetch the upstream branch list.
```

Comment:
@javiereguiluz Adding a new remote does not fetch anything. Thus, the `checkout` command results in an error like this:

```
fatal: Cannot update paths and switch to branch 'improve_install_chapter' at the same time.
Did you intend to checkout 'upstream/2.7' which can not be resolved as commit?
```